learn/tanssi/appchain-services/block-production.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Block Production Services
+description: Tanssi provides block production services allowing developers to deploy decentralized networks (appchains) with Ethereum-grade security through Symbiotic
+---
+
+# Block Production Services {: #block-production-services }
+
+## Introduction {: #introduction }
+
+As presented in the [Overview](/learn/tanssi/overview/){target=\_blank} article, Tanssi is an appchain infrastructure protocol that streamlines the deployment of blockchains with custom logic fitting a wide range of use cases, including DeFi, NFTs, Gaming, and any other use case development teams may want to address.
+
+Infrastructure poses a huge challenge for developers, requiring them to bootstrap sequencers, data preservers, and RPC endpoints, while also managing integrations, interoperability, and security. This demands valuable time and resources, diverting focus from what truly matters: delivering value to their users.
+
+Tanssi orchestrates resources, allowing developers to deploy decentralized networks (also known as actively validated services or AVSs) that are fully adaptable to any specific application or use case. In Tanssi terms, these application-specific blockchains are referred to as appchains. In this analogy, the Tanssi network resembles [Kubernetes](https://kubernetes.io){target=\_blank} in its role as an orchestrator, managing resources to guarantee the liveness and performance of the appchains.
+
+The protocol also tackles the security front by allowing appchains to select and connect to external security providers (like [Symbiotic](/learn/tanssi/external-security-providers/symbiotic/){target=\_blank}), ensuring Ethereum-grade security right from the start.
+
+This article covers the necessary aspects to consider when building and deploying your own modular blockchain, along with the most relevant technical aspects of the Tanssi protocol.
+
+## Block Production as a Service {: #block-production-as-a-service }
+
+The Tanssi protocol provides block production as a service, orchestrating a decentralized and trustless set of sequencers, ensuring the appchains' liveness. To do so, the protocol bridges both ends:
+
+- **Node operators** - who run sequencers, offering their block production services to get rewards
+- **Appchains** - which require transactions to be executed and included in blocks, keeping the network's liveness
+
```

Comment:
This part is a bit confusing. "To do so, the protocol bridges both ends" I'm unsure how this relates to the list below. It feels a bit repetitive tho as it is a bit the same but in a list.

Maybe just skip the bullet list and go to the next paragraph?

Reply:
It wasn't very clear, indeed. I changed it so both bridged ends are the actors node operators and develoeprs. wdyt?